pkg/setting/setting_test.go

```diff
@@ -30,6 +32,21 @@ func TestLoadingSettings(t *testing.T) {
 			So(cfg.RendererCallbackUrl, ShouldEqual, "http://localhost:3000/")
 		})
 
+		Convey("default.ini should have no semi-colon commented entries", func() {
+			file, err := os.Open("../../conf/defaults.ini")
+			if err != nil {
+				t.Error("failed to load defaults.ini file")
+			}
+			defer file.Close()
+
+			scanner := bufio.NewScanner(file)
+			for scanner.Scan() {
+				if strings.HasPrefix(scanner.Text(), ";") {
```

Comment:
Is it just out convention to comment entries with `;` and comments with `#` or is that in the ini format. It seems to be working both ways for me.

Reply:
I think just a convention. So commenting out the value with # would not be caught. But then I would have to somehow distinguish between a line with `#` that is supposed to be a comment - but maybe there is a reasonable way to do that using the iniFile structure and with text scanning?

Reply:
Yeah no idea, I think it is not a big issue and no need to handle it, would probably just add a comment that it assumes that only values are commented with `;` and not with `#`.